docs/01-get-started/03-working-with-the-database.md

```diff
@@ -137,12 +139,11 @@ class RecipeEndpoint extends Endpoint {
         'of the recipe in the first line, and then the instructions. The recipe should be easy '
         'to follow and include all necessary steps. Please provide a detailed recipe.';
 
-    final response = await gemini.generateContent([Content.text(prompt)]);
+    final response = await agent.send(prompt);
+    final responseText = response.output;
 
-    final responseText = response.text;
-
-    // Check if the response is empty or null
-    if (responseText == null || responseText.isEmpty) {
+    // Check if the response is empty
```

Comment:
Comments are inconsistent between files, double check that everything is consistent.

Reply:
Resolved